### PR TITLE
Fix misplaced ripple effect on various Android devices

### LINKED ISSUE #617: [Home] Misplaced ripple effect
## Version - 0.9.5

- Type of phone: Android 
- OS Version: 9.0, 10.0
- Model: Samsung A50, Samsung Galaxy S10

## Steps to reproduce

1. Complete onboarding procedure
2. Reach the home

## Expected Behavior

The ripple effect should be centered in the center of the icon.
Instead it results a bit shifted down.
The issue applies to each language.

Screenshot_20200422-002152_COVID Safe Paths

### PR DESCRIPTION
**Pull request by vestial**
Fixed misplaced ripple effect. I've tested it on Galaxy S10, Samsung A50, XIAOMI MI 8, Moto G7, Pixel 2/ 3 XL. 

#### Linked issues:

Fixes #617 

#### Screenshots:
Screenshot_1588253111

#### How to test

Check the home page of the app.

### CODE REVIEW

File: app/views/LocationTracking.js [hunk 1]
```diff
@@ -304,6 +305,20 @@ class LocationTracking extends Component {
     );
   }
 
+  //Change pulse location on a few problematic devices.
+  getPulseStyle() {
```
Comment (Patrick-Erichsen) [inline app/views/LocationTracking.js:309]: We haven't done a great job of documenting our [style guides](https://github.com/tripleblindmarket/covid-safe-paths/wiki/Code-styleguide#use-switch-statements), but one thing we've been trying to keep consistent is the usage of switch statements for blocks of logic with more than one or two `else if`s. Could you refactor to something more like this?

```js
getPulseStyle() {
  switch(height) {
    case height > 900:
      return { top: '29%' };
  }
}

**Comment by Patrick-Erichsen** [comment]
Hey @vestial , thanks for contributing!

So, when I run an emulator for `Galaxy S10` or `Moto G7` on the `develop` branch, I'm not having any issue with the pulse location. E.g. for the Galaxy S10

```
Display: 6.1 inches (~88.3% screen-to-body ratio)
Resolution: 1440 x 3040 pixels (~550 ppi density)
```

Can you confirm that you are still seeing the issue when running the `develop` branch on either of those devices? It doesn't look like there were any new commits since you forked that changed any styles on `LocationTracking`.

**Comment by vestial** [comment]
Hi @Patrick-Erichsen , thanks for checking it up! 

I currently don't see any issue in the `develop` branch with the pulse location for all the devices mentioned. It's weird because the issue happened to me with the old fork. I guess we can close the pull request and also the issue #617 .

**Comment by Patrick-Erichsen** [comment]
Yeah, that is weird - as I said, I don't see any new commits that would have fixed the issue.Thanks for taking a look either way though. 

Are you in our Slack channel yet? I don't think I have the permissions to add you, but if you visit the [volunteer intake form](https://docs.google.com/forms/d/e/1FAIpQLSdzqAxrlrxb_HqLh1KXnfPu1rse4aByS2krL1OYlN3qKChyqA/viewform) you should be added shortly. Would definitely appreciate any ongoing help 👍

**Comment by vestial** [comment]
I just got in the Slack channel recently. I would be glad to help!

**Pull request closed without merging**